assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSelectionPanel/Footer.js

```diff
@@ -92,23 +101,50 @@ export default function Footer( { isOpen, closePanel, savedItemSlugs } ) {
 		);
 	}
 
+	const [ dismissedItemsError, setDismissedItemsError ] = useState( null );
+
 	const saveSettings = useCallback(
 		async ( configuredAudiences ) => {
-			const { error } = await saveAudienceSettings( {
+			setDismissedItemsError( null );
+
+			let { error } = await saveAudienceSettings( {
 				configuredAudiences,
 			} );
 
+			if ( ! error ) {
+				const unselectedHiddenTileDismissedItems =
+					hiddenTileDismissedItems?.filter( ( item ) => {
+						const audienceResourceName = item.replace(
+							'audience-tile-',
+							''
+						);
+						return ! configuredAudiences.includes(
+							audienceResourceName
+						);
+					} );
+
+				if ( unselectedHiddenTileDismissedItems?.length > 0 ) {
+					( { error } = await removeDismissedItems(
+						...unselectedHiddenTileDismissedItems
+					) );
+
+					if ( error ) {
+						setDismissedItemsError( error );
+					}
+				}
+			}
+
 			return { error };
 		},
-		[ saveAudienceSettings ]
+		[ hiddenTileDismissedItems, removeDismissedItems, saveAudienceSettings ]
 	);
 
 	return (
 		<SelectionPanelFooter
 			savedItemSlugs={ savedItemSlugs }
 			selectedItemSlugs={ selectedItems }
 			saveSettings={ saveSettings }
-			saveError={ saveError }
+			saveError={ saveError || dismissedItemsError }
```

Comment:
It wasn't captured during definition, but seeing as we're handling the error from `saveAudienceSettings()`, we should handle the error from `removeDismissedItems()` in a similar manner.

Reply:
Thanks for handling this!